templates/nginx-plus-app-protect-ubnt1804-dev.template.yaml

```diff
@@ -541,7 +541,7 @@ Resources:
   AppProtectTargetGroup:
     Type: AWS::ElasticLoadBalancingV2::TargetGroup
     Properties:
-      Name: !Sub '${AWS::StackName}-AppProtectTargetGroup'
+      Name: !Sub '${AWS::StackName}-APTG'
```

Comment:
What was the reason to shorten this name?

Reply:
This change is submitted as the stack is not able to launch due to the below error.
maximum of 32 characters exceed for Targer Group

As taskcat is adding the stack name as tCat-<project-name>-<target-group-name> this complete name is exceeding 32 characters and hence I've modified the Target Group Name.